src/components/forms/Tagger.jsx

```diff
@@ -0,0 +1,84 @@
+import React, {PropTypes} from 'react';
+import Radium from 'radium';
+
+import TaggerRemoveComponent from './TaggerRemoveComponent';
+
+var ReactTags = require('react-tag-input').WithContext;
+
+@Radium
+export default class Tagger extends React.Component {
+
+  static propTypes = {
+    type: PropTypes.any.isRequired,
+    id: PropTypes.number.isRequired,
+    tags: PropTypes.array,
+    tagList: PropTypes.array,
+    freeform: PropTypes.bool
+  }
+
+  getDefaultProps() {
+    return {
+      tags: [],
+      tagList: [],
+      freeform: false
+    };
+  }
+
+  componentWillMount() {
+    this.setState({
+      tags: [ {id: 1, text: 'Apples'} ],
+      suggestions: ['Banana', 'Mango', 'Pear', 'Apricot', 'April', 'Appreciation']
+    });
+  }
+
+  handleDelete(i) {
+    var tags = this.state.tags;
```

Comment:
You want to avoid mutating the state directly in React => [Component API](http://facebook.github.io/react/docs/component-api.html) (scroll down to the warning)

A better way to do what you're trying to active would be
```
var tags = this.state.tags.concat([]);
```
This way you're copying the array instead of referencing it. This way someone else doesn't come along and create a weird `state` condition.